Root/OptimizationDump.cxx

```diff
@@ -514,7 +523,7 @@ EL::StatusCode OptimizationDump :: execute ()
         std::vector< ElementLink< xAOD::IParticleContainer > > constitLinks;
         if(jet->getAttribute("constituentLinks", constitLinks)) m_largeR_nsj[jetIndex] = constitLinks.size();
         // top tagging
-        m_largeR_topTag_veryloose[jetIndex] = VD::topTag(jet, "VeryLoose");
+        m_largeR_topTag_veryloose[jetIndex] = VD::topTag(jet, "VeryLoose", 2.0, 300.);
```

Comment:
Perhaps you want this to be 300. by default in the header? But I like to make it explicit.